### ISSUE TITLE
Empty folder inside mounted drive and "rclone config" connection refused

### ISSUE THREAD

**Issue by andrizmitnick** [Magisk-Modules-Repo/com.piyushgarg.rclone#8]
1. I had 4 remote connection on my rclone.conf and without any custom params
```
root@android:/storage/emulated/0 # rclone remount                
Default CACHEMODE writes                               
                                                       
[gdrive_personal] available at: -> [/mnt/cloud/gdrive_personal]
                                                       
[gsuite] available at: -> [/mnt/cloud/gsuite]          
                                                       
[office_365] available at: -> [/mnt/cloud/office_365]  
                                                       
[onedrive_personal] available at: -> [/mnt/cloud/onedrive_persona
l]                                                     
                                                       
Notice: /mnt/cloud served via HTTP at: http://127.0.0.1:38762
Notice: /mnt/cloud served via FTP at: ftp://127.0.0.1:38763
                                                       
...done                                                
```
2. 4 remote is mounted in /mnt/cloud/<remote_name>, but no contents at all inside each of them
![Screenshot_20190712063141-270x540](https://user-images.githubusercontent.com/21377475/61092308-f6c5e400-a46f-11e9-8a57-84c054150154.jpg)

3. Trying to add new remote with "rclone config" isn't working, it gives this error (below is auto yes and using auto no also doesn't work)
```
Remote config
Use auto config?
 * Say Y if not sure
 * Say N if you are working on a remote or headless machine
y) Yes
n) No
y/n> y
If your browser doesn't open automatically go to the following li
nk: http://127.0.0.1:53682/auth
Log in and authorize rclone for access
Waiting for code...
Got code
2019/07/11 23:27:54 Failed to configure token: failed to get toke
n: Post https://accounts.google.com/o/oauth2/token: dial tcp: loo
kup accounts.google.com on [::1]:53: read udp [::1]:48836->[::1]:
53: read: connection refused
```

**Comment by Geofferey** [comment]
Have you tried generating config using PC? There might be something lacking
in the binary... I also haven't set things up to export config to SD yet.

**Comment by Geofferey** [comment]
Have you tried generating config using PC? There might be something lacking in the binary... I also haven't set things up to export config to SD yet.

`[::1]:53: read udp [::1]:48836->[::1]`

^ this don't look right. Have you tried over wifi?

**Comment by andrizmitnick** [comment]
@Geofferey hi, yes my config was generated from my pc. using wifi/data or + vpn doesn't fix that.

btw are you able to reproduce my issue? i was thinking to backup and reflash clean rom on my device.

**Comment by imtyg** [comment]
@Geofferey @andrizmitnick FYI I have the exactly same problem. I tried to use the rclone binary from termux repo (the 39.59MB one) to replace it and it just works.

**Comment by andrizmitnick** [comment]
@imtyg oh wow, it does work!! why i didn't think of this 😂 lol. thanks mate. @Geofferey btw should i keep this issue open or just close for now?

**Comment by piyushgarg** [comment]
@imtyg share the direct link

**Comment by imtyg** [comment]
@piyushgarg https://dl.bintray.com/termux/termux-packages-24/aarch64/:rclone_1.48.0-1_aarch64.deb
You can also find out the x86 package.

**Comment by piyushgarg** [comment]
Try this build and let me know. https://beta.rclone.org/v1.48.0/testbuilds/rclone-android-16-arm.
It works for me.

**Comment by Geofferey** [comment]
The new version will include binaries for android that I compiled using termux to fix this issue. If I can find a way to compile binaries for x86/x64 droid using termux I will but currently don't have access to x86 droid machine. 

Here's a link to the binaries that will be included in 1.5
https://anonymousfiles.io/pgS4WXh5/
